test/unit/connection_string.test.ts

```diff
@@ -748,43 +747,6 @@ describe('Connection String', function () {
     });
   });
 
-  describe('feature flags', () => {
-    it('should be stored in the FEATURE_FLAGS Set', () => {
-      expect(FEATURE_FLAGS.size).to.equal(3);
-      expect(FEATURE_FLAGS.has(Symbol.for('@@mdb.skipPingOnConnect'))).to.be.true;
-      expect(FEATURE_FLAGS.has(Symbol.for('@@mdb.enableMongoLogger'))).to.be.true;
-      expect(FEATURE_FLAGS.has(Symbol.for('@@mdb.internalLoggerConfig'))).to.be.true;
-      // Add more flags here
-    });
-
-    it('should should ignore unknown symbols', () => {
-      const randomFlag = Symbol();
-      const client = new MongoClient('mongodb://iLoveJavaScript', { [randomFlag]: 23n });
-      expect(client.s.options).to.not.have.property(randomFlag);
-    });
-
-    it('should be prefixed with @@mdb.', () => {
-      for (const flag of FEATURE_FLAGS) {
-        expect(flag).to.be.a('symbol');
-        expect(flag).to.have.property('description');
-        expect(flag.description).to.match(/@@mdb\..+/);
-      }
-    });
-
-    it('should only exist if specified on options', () => {
-      const flag = Array.from(FEATURE_FLAGS)[0]; // grab a random supported flag
-      const client = new MongoClient('mongodb://iLoveJavaScript', { [flag]: true });
-      expect(client.s.options).to.have.property(flag, true);
-      expect(client.options).to.have.property(flag, true);
-    });
-
-    it('should support nullish values', () => {
```

Comment:
Can we add in testing to check that the `___` properties support null-ish values, specifically for `__enableMongoLogger`? I don't see any tests that check what happens when `__enableMongoLogger` flag is `null.`  l